src/Data/Profunctor/Mapping.hs

```diff
@@ -34,16 +37,36 @@ class (Traversing p, Closed p) => Mapping p where
   -- 'dimap' 'Data.Functor.Identity.Identity' 'Data.Functor.Identity.runIdentity' '.' 'map'' ≡ 'id'
   -- @
   map' :: Functor f => p a b -> p (f a) (f b)
+  map' = roam collect
+
+  roam :: (forall f. (Distributive f, Applicative f)
```

Comment:
We need the `Applicative` to implement `wanderMapping` and for several of the `Mapping` instances. If we had `Representable` instead of `Distributive`, I think we could use `Data.Functor.Rep.Co` to get everything we need.